ui_testing/pages/base_pages.py

```diff
@@ -645,3 +645,12 @@ def close_connection_with_database(self, db):
     def get_current_year(self):
         current_year = datetime.datetime.now()
         return str(current_year.year)
+
+    def Clicking_on_checkboxes_in_configure_table(self,field_one,field_two,child=False):
+        self.open_configure_table()
+        if child:
+            self.base_selenium.click(element='general:configure_child_table')
+        self.base_selenium.click(element=field_one)
```

Comment:
replace it with for loop :
```
for field in feilds:
       check first that feild is selected?
       if selected :
                   self.base_selenium.click(element=field_two)


```
you can use` checkbox_value.is_selected()` to make sure checkbox is selected or not
and                 checkbox_value = "//li[@id="materialType"']//input[@type='checkbox']"

Reply:
done but the  is selected didn't work always returns False